src/resolvers/helpers.ts

```diff
@@ -28,38 +28,43 @@ export function getParams(
             globalValidate,
             paramInfo.validate,
           );
+
         case "arg":
           return validateArg(
             convertArgToInstance(paramInfo, resolverData.args),
             paramInfo.getType(),
             globalValidate,
             paramInfo.validate,
           );
+
         case "context":
-          if (paramInfo.propertyName) {
-            return resolverData.context[paramInfo.propertyName];
-          }
+          if (paramInfo.propertyName) return resolverData.context[paramInfo.propertyName];
           return resolverData.context;
-        case "root":
-          // eslint-disable-next-line no-case-declarations
+
+        case "root": {
           const rootValue = paramInfo.propertyName
             ? resolverData.root[paramInfo.propertyName]
             : resolverData.root;
-          if (!paramInfo.getType) {
-            return rootValue;
-          }
+
+          if (!paramInfo.getType) return rootValue;
```

Comment:
I would like to see eslint rule preventing one-line if statement, i prefer explicit braces